simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/samplebuilder/impl/CustomMappingSampleBuilder.java

```diff
@@ -0,0 +1,110 @@
+package io.prometheus.client.dropwizard.samplebuilder.impl;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.dropwizard.samplebuilder.SampleBuilder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Custom {@link SampleBuilder} implementation to allow Dropwizard metrics to be translated to Prometheus metrics including custom labels and names.
+ * Prometheus metric name and labels are extracted from the Dropwizard name based on the provided list of {@link MapperConfig}s.
+ * The FIRST matching config will be used.
+ * If no config is matched, the {@link DefaultSampleBuilder} is used.
```

Comment:
What if the user wants unmatches metrics to be dropped?

Reply:
That can be another nice feature that would make this closer to the Graphite/StatsdExporter